entity-service/config/detekt/baseline.xml

```diff
@@ -3,24 +3,29 @@
   <ManuallySuppressedIssues></ManuallySuppressedIssues>
```

Comment:
don't systematically add all the issues in the baseline (if you do so, the tool becomes useless)

Reply:
Ok I understood